### PR TITLE
V1.1.0

### PR DESCRIPTION
**Pull request by Milstein**

### CODE REVIEW

**Review by joachimweyl — COMMENTED** [review]
Should we follow MD line length of 80 char. Answer was no.

File: EULA.md [hunk 1]
```diff
@@ -1,336 +1,139 @@
...
 
-You acknowledge and agree that the Services will collect and transmit
-telemetry data to MOC Alliance and its higher education, non-profit and
-industry MOC Alliance Collaborators. MOC Alliance and its MOC Alliance
-Collaborators may use telemetry data to operate and maintain (including
-to improve) the Services and for scientific research purposes.
+Telemetry data, including personal identifiable information, may be used and shared by Provider and the MOC Alliance Collaborators. Provider and the MOC Alliance Collaborators may use telemetry data, including personal identifiable information, for a variety of purposes including operating, maintaining and improving the Services, research (including for peer review, or other similar purposes), and for any purpose for which you may consent. Provider, MOC Alliance Collaborators, or a research organization may also publicly post and make broadly available for download telemetry data in an anonymized manner on the Internet.
```
Comment (joachimweyl) [inline EULA.md:97]: Is this saying the same thing in the first two sentences? Do we need the first sentence?

File: EULA.md [hunk 2]
```diff
@@ -1,336 +1,139 @@
...
-listed in the [list of providers](#providers). The Provider will take
-reasonable steps to arrange for you to retrieve such content, provided
-retrieval does not require an unreasonable amount of Provider's time or
-resources and that the content is not illegal; is not unlawfully pornographic
-or indecent; does not depict extreme acts of violence; does not advocate
-bigotry or hatred against any person or group of people based on their race,
-religion, ethnicity, sex, gender identity, sexual preference, disability, or
-impairment; and does not violate the privacy or infringe the rights of others).
-Any such retrieval shall be at your own expense.
+This Agreement may be terminated by you at any time by discontinuing your use of the Service and deleting your User Content or by written notice to the Provider’s technical/support contact listed here. In such an event, the Provider may immediately delete your account, but it may take up to four (4) weeks to do so. You should ensure prior to termination that you have transferred all data stored on the Services to another storage medium.
```
Comment (joachimweyl) [inline EULA.md:113]: Does this mean we always need to delete all user data within 4 weeks of them requesting termination?

**Review by joachimweyl — CHANGES_REQUESTED** [review]
Change ideas inline.

File: EULA.md (hunk 2 above)
Comment (Milstein) [reply EULA.md:113]: @msdisme : ^^

File: EULA.md (hunk 1 above)
Comment (msdisme) [reply EULA.md:97]: Worth exploring for a future version, but should be good for now (it's as written by OGC)

File: EULA.md (hunk 2 above)
Comment (msdisme) [reply EULA.md:113]: Yes, increased from 10 days in prior version

**Review by joachimweyl — APPROVED** [review]

**Review by msdisme — APPROVED** [review]
Looks good, Also had OGC review and they gave it a thumbs up.

**Pull request merged**